modules/skunk/src/test/scala/SkunkDatabaseSuite.scala

```diff
@@ -19,15 +18,19 @@ import edu.gemini.grackle.sql.test._
 trait SkunkDatabaseSuite extends SqlDatabaseSuite {
 
   // Slow because each usage will open a new socket, but ok for now.
-  lazy val pool: Resource[IO, Session[IO]] =
+  lazy val pool: Resource[IO, Session[IO]] = {
```

Comment:
We can replace this with a "resource fixture" that is shared for the entire suite in a follow-up PR.